frontend/src/components/grantaccess/GrantDelegatedAdmin.vue

```diff
@@ -0,0 +1,222 @@
+<script setup lang="ts">
+import { ref, type PropType } from 'vue';
+import router from '@/router';
+import { number, object, string } from 'yup';
+import { ErrorMessage, Field, Form as VeeForm } from 'vee-validate';
+
+import { UserType } from 'fam-app-acsctl-api';
+
+import Dropdown from 'primevue/dropdown';
+import Button from '@/components/common/Button.vue';
+import { IconSize } from '@/enum/IconEnum';
+
+import { isLoading } from '@/store/LoadingState';
+import type { FamRoleDto } from 'fam-admin-mgmt-api/model';
+
+const props = defineProps({
+    delegatedRoleOptions: {
```

Comment:
Hi Nick, so here we don't need to set this value from the router anymore, if you check the `GrantApplicationAdmin.vue` file,  we now define and get its value inside the component, something like below:
<img width="421" alt="Screen Shot 2024-02-26 at 2 14 51 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/c712067f-0473-4896-92b8-714671ce9cee">

Reply:
And I think we only assign abstract role with forest client numbers for now, ignore the concrete role, but let's ask Olga tomorrow to confirm.

Reply:
Hi Catherine. We can leave it unfiltered for now, right? @MCatherine1994

Reply:
yeah, no need to filter it. But we don't need to set this value from the router, check the `GrantApplicationAdmin.vue` file, and we now define and get the role options value inside the component

Reply:
Thanks, I'll update it as suggested.